package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "chartjs-plugin-annotation",
   "description": "Annotations for Chart.js",
-  "version": "0.5.3",
+  "version": "0.5.4",
```

Comment:
Please revert all changes to this file, we will build it during the release process.